android/src/main/java/com/reactnativecommunity/imageeditor/ImageEditorModuleImpl.kt

```diff
@@ -434,6 +431,20 @@ class ImageEditorModuleImpl(private val reactContext: ReactApplicationContext) {
             )
 
         // Utils
+        private fun getMimeType(outOptions: BitmapFactory.Options, format: String?): String {
+            val mimeType =
+                when (format) {
+                    "webp" -> "image/webp"
+                    "png" -> "image/png"
+                    "jpeg" -> "image/jpeg"
```

Comment:
How about keeping those values in static consts?

Reply:
good idea!